### DISCUSSION TITLE
Is there a bounty system in place for new features and bugs?

### DISCUSSION THREAD

**Discussion by amirHizkiya** [YOURLS/YOURLS#3586] [Ideas]
I'd like to donate to certain cause, or toward a bank of causes that may be listed in the roadmap. 

If not, maybe this something worth investing in?

**Comment by ozh** [comment]
Donations are welcome, as a way to say thanks for what already exists

Donations to help develop particular features are also welcome, but in a different currency : your own free time :)

**Comment by amirHizkiya** [comment]
Yeah I get that, and it makes sense. 
I just thought that in the long run it would help with a more consistent roadmap and pipeline although I have not been a part of a project like this before so I might be wrong. 

Just an idea that I thought could help get the ball rolling faster on some features.

**Comment by amirHizkiya** [comment]
I see other projects poping up and developing at a pretty fast pace (opensource as well), I would hate for this project to loose steam. On the contrary, I think this project could have something special here with the WP structure of things (hooks/plugins/etc.). 

If by time you are referring to project management, and if a bounty system is something this project is interested in, I can help get the ball rolling on that.

Thanks for the cool project.
